sdk/evaluation/azure-ai-evaluation/azure/ai/evaluation/_evaluate/_evaluate.py

```diff
@@ -36,13 +36,47 @@
 
 TClient = TypeVar("TClient", ProxyClient, CodeClient)
 
+# For metrics (aggregates) whose metric names intentionally differ from their
+# originating column name, usually because the aggregation of the original value
+# means something sufficiently different.
+# Note that content safety metrics are handled seprately.
+METRIC_COLUMN_NAME_REPLACEMENTS = {
+    "groundedness_pro_label": "groundedness_pro_passing_rate",
+}
+
 
 class __EvaluatorInfo(TypedDict):
     result: pd.DataFrame
     metrics: Dict[str, Any]
     run_summary: Dict[str, Any]
 
 
+def _aggregate_other_metrics(df: pd.DataFrame) -> Tuple[List[str], Dict[str, float]]:
```

Comment:
Is it possible to converge `_aggregate_other_metrics` and `_aggregate_content_safety_metrics`? If it's complex, let's create a task and refactor later.

Reply:
I initially considered that, but I'd rather not.

The content safety aggregation relies on pattern-matching content safety evaluator names to metric prefixes. 

The new aggregation I added is a much more naive metric-name matcher. They have similar effects, but very different methods to achieve them.